File: String/(LEETCODE)LengthOfLastWord.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
       int ans=0;
       for(int i=s.size()-1;i>=0;i--)
       {
           if(s[i]==32 && ans>0)
               return ans;
           if(s[i]!=32)
               ans++;
       }
        return ans;

    }
};

File: String/(LEETCODE) Length of Last Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        i = len(s) - 1
        length = 0

        while s[i] == " ":
            i = i-1
        while i >= 0 and s[i] != " ":
            length = length+1
            i = i-1

        return length
